src/odemis/gui/model/__init__.py

```diff
@@ -936,6 +936,7 @@ def __init__(self, name, stage=None, stream_classes=None, fov_hw=None):
             # the stage
             pos = self.stage_pos.value
             view_pos_init = (pos["x"], pos["y"])
+            self.stage_is_moving = BooleanVA(False)
```

Comment:
You cannot put this on every view. You need a global boolean. Either on the tab model or the GUI model (like `is_preparing` and `is_acquiring`). I'd suggest to just move it to the GUI model. Now, that means that all views must keep a reference to the main GUI, which is a little of a pity, but should be fine.
Also as a side note, one could argue that a boolean is a little dangerous, because if for any reason, the end of the move doesn't set it back to False, Odemis might think it's moving forever (until the next move). A safer content would be to store the future corresponding to the move (as it's possible to check that the future is done or not). Let's just keep this in mind for now, and keep it a boolean.